resources-domain/resources-import-producer/tests/unit/handler.test.ts

```diff
@@ -18,9 +18,9 @@ import parseISO from 'date-fns/parseISO';
 import { handler, HttpError, isHttpError, KhpApiResource, KhpApiResponse } from '../../src';
 import each from 'jest-each';
 // eslint-disable-next-line prettier/prettier
```

Comment:
While trying to figure out what was wrong with our linter that didn't catch the [non-standard objects-in-an-array line break](https://github.com/techmatters/hrm/pull/406/files#diff-fc1f61c7e3c4e1bc76ef9e8ce3e1d4730b21ceda7b7f2f6b4d07185a0ece6dd7R143), I found that this disable line suppresses the error, but breaks prettier processing for the rest of the file. Removing the `import type` causes the file to be properly prettified.

This is more informational than a part of the PR review.

We've needed to update prettier for forever. I never got around to it after I pulled it out of one of my initial giant PRs in my first couple of months.

I'm going to open a new PR that fixes prettier so we stop running into these weird behaviors.